specification/resources/resource-manager/Microsoft.Resources/preview/2023-07-01-preview/changes.json

```diff
@@ -0,0 +1,498 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "targetResourceType": {
+          "description": "The namespace and type of the resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "changeType": {
+          "description": "The type of change that was captured in the resource",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Update",
+            "Delete",
+            "Create"
+          ],
+          "x-ms-enum": {
+            "name": "ChangeType",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Update",
+                "description": "An existing resource underwent a change",
+                "name": "Update"
+              },
+              {
+                "value": "Delete",
+                "description": "An existing resource was deleted",
+                "name": "Delete"
+              },
+              {
+                "value": "Create",
+                "description": "A newly created resource",
+                "name": "Create"
+              }
+            ]
+          }
+        },
+        "changeAttributes": {
+          "$ref": "#/definitions/ChangeAttributes"
+        },
+        "changes": {
+          "$ref": "#/definitions/ChangesDictionary"
+        }
+      }
+    },
+    "ChangeAttributes": {
+      "description": "Details about the change resource",
+      "type": "object",
+      "properties": {
+        "correlationId": {
+          "description": "The ARM correlation ID of the change resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "timestamp": {
+          "description": "The time the change(s) on the target resource ocurred",
+          "type": "string",
+          "readOnly": true
+        },
+        "changesCount": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The number of changes this resource captures"
+        },
+        "previousResourceSnapshotId": {
+          "description": "The GUID of the previous snapshot",
+          "type": "string",
+          "readOnly": true
+        },
+        "newResourceSnapshotId": {
+          "description": "The GUID of the new snapshot",
+          "type": "string",
+          "readOnly": true
+        },
+        "isTruncated": {
+          "description": "The flag indicating whether property changes dictionary was truncated in this resource",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "changedBy": {
+          "description": "The entity that made the change(s)",
+          "type": "string",
+          "readOnly": true
+        },
+        "changedByType": {
+          "description": "The entity type that made the change(s)",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
```

Comment:
>enum [](http://example.com/codeflow?start=11&length=4)

add 'Unknown' , since we updated the code to set to this value when changedBy is not-empty, but this is empty.